webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -228,19 +142,84 @@ const useModalCreateGroup = ({ onClose, users, group }: ModalProps): ModalHooks
     }
   }, [form, onClose, group, canModifyPermissions, fetchGroupRoles, groupRoles]);
 
-  const modalOpen = useCallback(() => {
-    openOrUpdate({
-      closable: true,
-      content: <ModalForm form={form} group={group} groupRoles={groupRoles} users={users} />,
-      icon: null,
-      okText: group ? MODAL_HEADER_LABEL_EDIT : MODAL_HEADER_LABEL_CREATE,
-      onCancel: handleCancel,
-      onOk: onOk,
-      title: <h5>{group ? MODAL_HEADER_LABEL_EDIT : MODAL_HEADER_LABEL_CREATE}</h5>,
-    });
-  }, [form, handleCancel, onOk, openOrUpdate, users, group, groupRoles]);
-
-  return { modalOpen, ...modalHook };
+  return (
```

Comment:
i dont remember how it looked like but should the text fields be full width?

<img width="383" alt="image" src="https://user-images.githubusercontent.com/109113616/229241001-4e4c9ad8-58f0-4b6f-af24-ef47e2db1a2d.png">

Reply:
thanks, had unnecessary `labelCol` and `wrapperCol`